src/components/modals/sell-modal/index.js

```diff
@@ -6,14 +6,15 @@ import Button from "@ui/button";
 import { validate, Network } from "bitcoin-address-validation";
 import InputGroup from "react-bootstrap/InputGroup";
 import Form from "react-bootstrap/Form";
-import { TESTNET, ORDINALS_EXPLORER_URL } from "@lib/constants";
+import { TESTNET, ORDINALS_EXPLORER_URL, NOSTR_SELL_KIND_INSCRIPTION } from "@lib/constants.config";
```

Comment:
I think it's best to keep updates to buy/sell modals in a separate PR for complexity, wdyt?